moveit_ros/moveit_servo/src/servo_parameters.yaml

```diff
@@ -0,0 +1,261 @@
+    type: string,
+    default_value: "~/delta_twist_cmds",
+    description: "The topic on which servo will receive the twist commands"
+  }
+
+  joint_command_in_topic: {
+    type: string,
+    default_value: "~/delta_joint_cmds",
+    description: "The topic on which servo will receive the joint jog commands"
+  }
+
+  robot_link_command_frame: {
+    type: string,
+    default_value: "panda_link0",
+    description: "The frame of incoming command, change this to your own value when using a different robot."
+  }
+
+  command_in_type: {
+    type: string,
+    default_value: "unitless",
+    description: "The unit of the incoming command.",
+    validation: {
+      one_of<>: [["unitless", "speed_units"]]
+    }
+  }
+
+  linear_scale: {
+    type: double,
+    default_value: 0.4,
+    description: "Max linear velocity. Unit is [m/s]. Only used for Cartesian commands."
+  }
+
+  rotational_scale: {
+    type: double,
+    default_value: 0.8,
+    description: "Max angular velocity. Unit is [rad/s]. Only used for Cartesian commands"
+  }
+
+  joint_scale: {
+    type: double,
+    default_value: 0.5,
+    description: "Max joint angular/linear velocity. Only used for joint commands on joint_command_in_topic"
+  }
+
+  override_velocity_scaling_factor: {
+    type: double,
+    default_value: 0.0,
+    description: "Scaling factor when servo overrides the velocity (eg: near singularities)"
+  }
+
+  command_out_topic: {
+    type: string,
+    default_value: "/panda_arm_controller/joint_trajectory",
+    description: "The topic on which servo will publish the joint trajectory \
+                  Change this to the topic your controller requires."
+  }
+
+  publish_period: {
+    type: double,
+    default_value: 0.034,
+    description: " 1 / (Nominal publish rate) [seconds]",
+    validation: {
+      gt<>: 0.0
+    }
+  }
+
+  command_out_type: {
+    type: string,
+    default_value: "trajectory_msgs/JointTrajectory",
+    description: "The type of command that servo will publish",
+    validation: {
+      one_of<>: [["trajectory_msgs/JointTrajectory", "std_msgs/Float64MultiArray"]]
+    }
+  }
+
+  publish_joint_positions: {
+    type: bool,
+    default_value: true,
+    description: "If true servo will publish joint positions in the output command"
+  }
+
+  publish_joint_velocities: {
+    type: bool,
+    default_value: true,
+    description: "If true servo will publish joint velocities in the output command"
+  }
+
+  publish_joint_accelerations: {
+    type: bool,
+    default_value: false,
+    description: "If true servo will publish joint accelerations in the output command"
+  }
+
+  joint_topic: {
+    type: string,
+    default_value: "/joint_states",
+    description: "The topic on which joint states can be monitored"
+  }
+
+  smoothing_filter_plugin_name: {
+    type: string,
+    default_value: "online_signal_smoothing::ButterworthFilterPlugin",
+    description: "The name of the smoothing plugin to be used"
+  }
```

Comment:
I recently learned that the smoothing filter coefficient was hard-coded in the code to be 1.5. Could you make this a parameter in this PR too, if that's possible?

Reply:
There is a PR going to make that a parameter .
https://github.com/ros-planning/moveit2/pull/2091
Is this what you are referring to ?

Reply:
@sea-bass  #2114  and #2091  have taken care of this.